app/src/main/kotlin/org/fossify/calculator/helpers/converters/VolumeConverter.kt

```diff
@@ -119,6 +119,13 @@ object VolumeConverter : Converter {
             factor = 0.00011829411825,
             key = "GillUS"
         )
+        
+        data object CupUS : Unit(
+            nameResId = R.string.unit_volume_cup_us,
+            symbolResId = R.string.unit_volume_cup_us_symbol,
+            factor = 0.00024,
```

Comment:
```suggestion
            factor = 0.0002365882365,
```

Let's use better precision factor, so conversion to milliliters is the same as stated on [Wikipedia](https://en.wikipedia.org/wiki/Cup_(unit)).

Reply:
This is a thing to argue, because we have a customary cup and a legal cup in US. I see in the old app there was 0.00024 factor.
It depends what we want (not saying about the imperial, metric etc).

Okay, I checked random converters online and the most common approach is to add at least the imperial cup too.
What do you think?

Reply:
Let's add both - customary and legal.

Reply:
@phoropter you are an expert, so you can drop a hint someday what's really used in the US